### PR TITLE
Fix Issue #26

### PR DESCRIPTION
**Pull request by jjl**
I hate jdbc. Resolves issue #26

### CODE REVIEW

**Comment by jjl** [comment]
I hate travis too. I don't get that error locally :/

**Comment by jjl** [comment]
And we're green!

File: build.boot [hunk 1]
```diff
@@ -0,0 +1,41 @@
+(set-env!
+  :version "1.2.0"
```
Comment (enragedginger) [inline build.boot]: Can you update the version number to 1.2.1 since 1.2.0 was already published?

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Note: this library was once called [jdbc-pg-sanity](https://clojars.org/jdbc-pg-
 
 ## Installation
 
-Add `mpg` as a leiningen dependency:
+Add `mpg` as a leiningen or boot dependency:
 
 ```clojure
 [mpg "1.2.0"]
```
Comment (enragedginger) [inline README.md:31]: Can you update the version number here as well?

**Review by enragedginger — COMMENTED** [review]
I tried version 1.2.0 with my project and I'm getting the same errors as before--likely because version 1.2.0 was already published. Can you set the version number to 1.2.1? At that point, I'll give this a shot again.

File: README.md (hunk 2 above)
Comment (jjl) [reply README.md:31]: Nope. Shane will do this as part of the release. It'll probably go out as 1.3 with some notes about how we should now support inserts properly.

**Comment by jjl** [comment]
I've pushed a bump to 1.2.1, but it will probably eventually go out as 1.3.0 (and shane will update the README with the correct version while he's doing the release)

**Comment by JuneKelly** [comment]
Looks good, planning to get this merged later tonight or tomorrow, and release as 1.3

Thanks @jjl, I owe you a pint.

**Comment by jjl** [comment]
They come in half litres over here. They always did in London too, it's just here they never sold them as pints ;)

**Pull request merged**